### PR TITLE
Update README for images

### PR DESCRIPTION
**Pull request by jhadvig**
Replaced STI -> S2I
Added rest of STI images
Fixed STI images docs url
Added DB images

### CODE REVIEW

**Comment by jhadvig** [comment]
@bparees @kargakis ptal

File: README.md [hunk 1]
```diff
@@ -163,10 +163,21 @@ FAQ
...
+    OpenShift is designed to run any existing Docker images.  In addition you can define builds that will produce new Docker images from a Dockerfile.  However the real magic of OpenShift can be seen when using [Source-To-Image](https://github.com/openshift/source-to-image)(S2I) builds which allow you to simply supply an application source repository which will be combined with an existing S2I-enabled Docker image to produce a new runnable image that runs your application.  We are continuing to grow the ecosystem of S2I-enabled images and documenting them [here](http://docs.openshift.org/latest/using_images/sti_images/overview.html). Our available images are:
```
Comment (mfojtik) [inline README.md]: Source-To-Image enabled

**Comment by mfojtik** [comment]
one nit, otherwise LGTM

@smarterclayton merge?

**Comment by kargakis** [comment]
Thanks for doing this! 

I am not sure about keeping the STI abbreviation even as S2I, otherwise LGTM.

**Comment by jhadvig** [comment]
@mfojtik updated, thx

**Comment by jhadvig** [comment]
@bparees thoughts ?

**Comment by mfojtik** [comment]
LGTM @smarterclayton merge?

**Comment by bparees** [comment]
works for me.

**Comment by smarterclayton** [comment]
Approved [merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**